_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+- Users with the `VIEWJOB` role option can now view all jobs, such as by using the `SHOW JOBS` command. This role can be revoked by granting the `NOVIEWJOB` role. [#96382][#96382]
+
+<h3 id="v23-1-0-alpha-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Jobs that utilize protected timestamp system (such as `BACKUP`, `CHANGEFEED`, or `IMPORT` now produce metrics that can be monitored to detect cases when a job leaves a stale protected timestamp that will prevent garbage collection from occurring. [#97148][#97148]
+- Changefeeds now automatically expire PTS records for paused jobs if the changefeed is configured with the `gc_protect_expires_after` option. [#97148][#97148]
+- UDFs can now be referenced from column DEFAULT expressions when creating a new table or issuing the `SET DEFAULT` command. Backup and restore operations also back up and restore UDF IDs that are referenced in a column's DEFAULT expression. If UDF dependencies are missing and the `skip_missing_udfs` flag is provided, the DEFAULT expressions are dropped during a restore operation. [#97501][#97501]
+- Error messages related to `initial_scan_only`, such as "cannot specify both initial_scan_only and resolved", n0.ow substitute in "initial_scan='only'".[#97487][#97487]
+
+<h3 id="v23-1-0-alpha-5-sql-language-changes">SQL language changes</h3>
+
+- String literals are now allowed for region names in DDL syntax, in addition to quoted syntax. [#97021][#97021]
+- It is now possible to use `*` inside `CREATE VIEW`. The list of columns is expanded at the time the view is created, so that new columns added after the view was defined are not included in the view. This behavior is the same as PostgreSQL. [#97515][#97515]
+- The default value of `sql.stats.cleanup.rows_to_delete_per_txn` has been increased to 10k to increase efficiency of the cleanup job for SQL statistics. [#97642][#97642]
+- The new new session setting `optimizer_use_improved_split_disjunction_for_joins` allows the optimizer to split disjunctions (`OR` expressions) in more `JOIN` conditions by building a `UNION` of two `JOIN` expressions. If this setting is true, all disjunctions in inner, semi, and anti `JOIN`s` will be split. Otherwise, only disjunctions that potentially contain an equijoin condition will be split. [#97696][#97696]
+- Builtins have been added for `tsvector`, `to_tsquery`, `phraseto_tsquery`, `plainto_tsquery`, which parse input documents into tsvectors and tsqueries respectively. The new `ts_parse` builtin is used to debug the text search parser. [#92966][#92966]
+- The new session variable `inject_retry_errors_on_commit_enabled` returns a transaction retry error if it is run inside of an explicit transaction when it is set to `true`. The transaction retry error continues to be returned until `inject_retry_errors_on_commit_enabled` is set to `false`. This setting allows developers to test their transaction retry logic. [#97226][#97226]
```

Comment:
Link to session vars table. Link to a txn retry error doc (maybe the error reference).
```suggestion
- The new session variable `inject_retry_errors_on_commit_enabled` returns a transaction retry error if it is run inside of an explicit transaction when it is set to `true`. The transaction retry error continues to be returned until `inject_retry_errors_on_commit_enabled` is set to `false`. This setting allows you to test your transaction retry logic. [#97226][#97226]
```